src/app/layout/layout.component.ts

```diff
@@ -47,7 +48,11 @@ export class LayoutComponent implements OnInit, OnDestroy, AfterViewInit {
     @Inject(DOCUMENT) private document: any,
     @Inject(PLATFORM_ID) private platformId: string,
     private kycFacadeService: KycFacadeService
-  ) {}
+  ) {
+    if(window.localStorage.getItem('phishing') != 'false'){
+      window.localStorage.setItem('phishing' , 'true') 
+    }
```

Comment:
Tu fais la vérification de la valeur de 'phishing' dans le stockage local et la définissez sur 'true' si elle n'est pas 'false'. Ce comportement pourrait être confus à première vue. Si vous voulez définir une valeur par défaut pour 'phishing', il serait plus clair de le faire lorsque la valeur est null ou undefined, plutôt que lorsqu'elle n'est pas 'false'

`if(window.localStorage.getItem('phishing') === null) {
  window.localStorage.setItem('phishing' , 'true') 
}
`

Reply:
Il est important de noter que l'accès au stockage local peut échouer si l'utilisateur a désactivé les cookies ou s'il utilise le mode privé sur certains navigateurs. Il serait donc préférable d'encapsuler cette logique dans un bloc try/catch pour gérer d'éventuelles erreurs
 
 `try {
  if(window.localStorage.getItem('phishing') === null) {
    window.localStorage.setItem('phishing' , 'true') 
  }
} catch (error) {
  console.error('Could not access localStorage:', error);
}
`

Reply:
how can i refactor this : 
constructor(){if(window.localStorage.getItem('phishing') != 'false'){
      window.localStorage.setItem('phishing' , 'true') 
    }}

Reply:
@ksibisamir done
https://github.com/SaTT-Wallet/Frontend/pull/316/commits/721a281491998a52a4d0bf2ea6800f140599b6d4